### PR TITLE
Explore for urlbar events

### PR DESCRIPTION
**Pull request by rebecca-burwei**
Checklist for reviewer:

When adding a new derived dataset:
- [x] Ensure that the data is not available already (fully or partially) and recommend extending an existing dataset in favor of creating new ones. Data may be available in [bigquery-etl repository](https://github.com/mozilla/bigquery-etl), [looker-hub](https://github.com/mozilla/looker-hub) or in [looker-spoke-default](https://github.com/mozilla/looker-spoke-default/tree/e1315853507fc1ac6e78d252d53dc8df5f5f322b).
- [x] Avoid merging a PR that includes the logic of a [core metric](https://docs.telemetry.mozilla.org/metrics/index.html) or complex business logic. The recommendation is to implement core business logic in bigquery-etl. E.g. The [type of search](https://github.com/mozilla/bigquery-etl/blob/a3e59f90326816a2ecaaa3e9d5b57fe9552f7d70/sql/moz-fx-data-shared-prod/search_derived/mobile_search_clients_daily_v1/query.sql#L781) or the [calculation of DAU or visited URIs](https://github.com/mozilla/bigquery-etl/blob/9bca48821a8a0d40b1700cc14ecd8068d132ed06/sql/moz-fx-data-shared-prod/telemetry_derived/firefox_desktop_exact_mau28_by_dimensions_v1/query.sql).
- [x] Avoid merging code in Looker Explores/Views that implement analysis with multiple lines of code or that will be likely replicated in the future. Instead, aim for extending an existing dataset to include the required logic, and use [Looker aggregates](https://cloud.google.com/looker/docs/aggregate_awareness) to facilitate the analysis.
- [x] Avoid merging a PR with logic that requires validation and health checks. It is recommended to implement it in bigquery-etl for full test coverage and failure alerts.

### CODE REVIEW

File: firefox_desktop/views/urlbar_events.view.lkml [hunk 1]
```diff
@@ -0,0 +1,503 @@
...
+  measure: addon_impressions {
+    group_label: "Addon"
+    sql: sum(case when ${num_add_on_impressions} > 0 and ${is_terminal} then 1 else 0 end);;
+    type: number
+  }
+
+  measure: addon_CTR {
+    group_label: "Addon"
+    description: "Clicks / Impressions"
+    sql: safe_divide(${urlbar_clicks}, ${urlbar_impressions});;
```
Comment (dzeber) [inline firefox_desktop/views/urlbar_events.view.lkml:194]: The variable names need to be updated here and in the CTR measure definitions below

File: firefox_desktop/views/urlbar_events.view.lkml [hunk 2]
```diff
@@ -0,0 +1,503 @@
...
+  datatype: date
+}
+
+  measure: event_count {
+    hidden: yes
+  }
+
+measure: urlbar_clicks {
+  group_label: "Urlbar Metrics"
+  sql: sum(case when ${is_terminal} and ${session_action_type} = "engaged" then 1 else 0 end);;
```
Comment (dzeber) [inline firefox_desktop/views/urlbar_events.view.lkml:156]: I recommend using `COUNTIF` rather than this pattern for conciseness and readability

**Review by dzeber — APPROVED** [review]
LGTM! I've left a couple of inline comments.

**Comment by rebecca-burwei** [comment]
@dzeber thanks for catching those! fixed

**Pull request merged**